internal/webhooks/machinedeployment.go

```diff
@@ -168,6 +164,10 @@ func (webhook *MachineDeployment) Default(ctx context.Context, obj runtime.Objec
 		m.Spec.Template.Spec.InfrastructureRef.Namespace = m.Namespace
 	}
 
+	if m.Spec.Template.Spec.MinReadySeconds == nil {
```

Comment:
Q: can we drop this now that we have // +default=0 on the field definition?

Reply:
We need to keep it because the +default=0 isn't used.

Reply:
Resolving here as we have it covered in other findings